### PR TITLE
En-6267: unit tests update/process

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: update/mock/pendingTransactionProcessorStrub.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package mock
```
Comment (sasurobert) [inline update/mock/pendingTransactionProcessorStrub.go:1]: rename file to Stub.

Comment (miiu96) [reply update/mock/pendingTransactionProcessorStrub.go:1]: done.

File: update/mock/transactionCoordinatorMock.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+	RequestBlockTransactionsCalled                       func(body *block.Body)
+	IsDataPreparedForProcessingCalled                    func(haveTime func() time.Duration) error
+	SaveBlockDataToStorageCalled                         func(body *block.Body) error
+	RestoreBlockDataFromStorageCalled                    func(body *block.Body) (int, error)
+	RemoveBlockDataFromPoolCalled                        func(body *block.Body) error
+	ProcessBlockTransactionCalled                        func(body *block.Body, haveTime func() time.Duration) error
+	CreateBlockStartedCalled                             func()
+	CreateMbsAndProcessCrossShardTransactionsDstMeCalled func(header data.HeaderHandler,
+		processedMiniBlocksHashes map[string]struct{},
+
```
[the remark this answers is not in the document]
Comment (miiu96) [reply update/mock/transactionCoordinatorMock.go:24]: done.

File: update/mock/txProcessorMock.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// TxProcessorMock -
+type TxProcessorMock struct {
+	ProcessTransactionCalled         func(transaction *transaction.Transaction) error
+	SetBalancesToTrieCalled          func(accBalance map[string]*big.Int) (rootHash []byte, err error)
+	ProcessSmartContractResultCalled func(scr *smartContractResult.SmartContractResult) error
+}
+
+// ProcessTransaction -
+func (etm *TxProcessorMock) ProcessTransaction(transaction *transaction.Transaction) error {
+	return etm.ProcessTransactionCalled(transaction)
```
[the remark this answers is not in the document]
Comment (miiu96) [reply update/mock/txProcessorMock.go:19]: done./

**Pull request merged**